modules/Microsoft.Sql/managedInstances/readme.md

```diff
@@ -300,7 +300,7 @@ module managedInstances './Microsoft.Sql/managedInstances/deploy.bicep' = {
   params: {
     administratorLogin: 'adminUserName'
     administratorLoginPassword: '<administratorLoginPassword>'
-    name: '<<namePrefix>>-sqlmicom'
+    name: '<<namePrefix>>-sqlmicom-001'
```

Comment:
As mentioned I'd suggest to hold on the rename at least until we will fix the issue on the test environment side